script.js

```diff
@@ -0,0 +1,40 @@
+function calculate() {
+    
```

Comment:
No need for the blank lines between variables and the top of the function or variables and each other.